### DISCUSSION TITLE
sskorol lib and test-data-supplier do not work after switching a project to Java 17

### DISCUSSION THREAD

**Discussion by da-oleg-ignatiev** [sskorol/test-data-supplier#109] [Ideas]
Hi. Can you please make sskorol lib work with Java 17. It keeps throwing an error once I switched my maven project to Java 17. All data supplier methods fail upon compilation. Do you have any ideas? Thank you.

**Comment by sskorol** [comment]
@da-oleg-ignatiev hi, I've just tried to upgrade one of my projects that uses test-data-supplier to OpenJDK 17.0.2, and it works as expected. So it seems like you have a very specific issue which depends on your configuration. Please share a minimal example to reproduce this issue on Java 17, and I'll take a look.
